### PR TITLE
Views

### PR DESCRIPTION
**Pull request by agneta1991**
# Within this branch i have:

- I have implemented the view of every page according to the project requirements
- Added Css to make it look nicer.
- Added additional information into data base to have better presentation of the project

**Be aware that posts are listed under the user by a number and its not the actual id of the post, where in the URL the id of the post must be used, as example: http://localhost:3000/users/3/posts/1 if that users post's id is not 1, it will throw an error**

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1,4 +1,32 @@
...
+<div class='user_bio'>
+  <%= @user.bio %>
+</div>
+
+<div class='posts'>
+  <% post_number = 1 %>
+  <% @posts.each do |post| %>
+    <div class='individual_post'>
+      <h2>Post #<%= post_number %></h2>
+      <h3><%= post.title %></h3>
```
Comment (ShadyShawkat) [inline app/views/users/show.html.erb]: - [ ] Kindly make sure to use the created methods in the model (`most_recent_post`), as per [this](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md#:~:text=Use%20methods%20that%20you%20have%20created%20in%20the%20Blog%20App%20%2D%20processing%20data%20in%20models.) requirement.

File: app/views/posts/index.html.erb [hunk 2]
```diff
@@ -1,9 +1,34 @@
...
+      </p>
+      <div class='comments'>
+        <p class='number_of_comments'> Comments: <%= post.comments_counter %> Likes: <%= post.likes_counter %></p>
+      </div>
+      <div class='comments_div'>
+        <h3> Comments: </h3>
+        <% post.comments.each do |comment| %>
+          <p><%= comment.author.name %>: <%= comment.text %></p>
+        <% end %>
+      </div>
```
Comment (ShadyShawkat) [inline app/views/posts/index.html.erb]: - [ ] Kindly make sure to use the created methods in the model (`most_recent_comments`), as per [this](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md#:~:text=Use%20methods%20that%20you%20have%20created%20in%20the%20Blog%20App%20%2D%20processing%20data%20in%20models.) requirement.

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @agneta1991, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by gemmen29 — APPROVED** [review]
Hi @agneta1991, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**